.github/workflows/beta_artifacts.yml

```diff
@@ -81,8 +81,12 @@ jobs:
         env:
           COMPILER: gcc
         run: ci/actions/linux/install_deps.sh
+      - name: Build Docker (nanocurrency/nano-beta)
+        run: TRAVIS_TAG=${TAG} ci/actions/linux/docker-build.sh
+        env:
+          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
```

Comment:
The build process doesn't need the DOCKER_PASSWORD

Reply:
It should be ok now.